jdbc/jooq/src/main/java/example/springdata/jdbc/jooq/JooqMethods.java

```diff
@@ -0,0 +1,46 @@
+package example.springdata.jdbc.jooq;
+
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static example.springdata.jdbc.basics.simpleentity.domain.tables.Category.CATEGORY;
+
+/**
+ * Implementations for custom repository access using jOOQ.
+ *
+ * @author Florian Lüdiger
+ */
+@Component
+public class JooqMethods {
```

Comment:
This shouldn't be an arbitrary component but a custom implementation for a repository method. See https://docs.spring.io/spring-data/jdbc/docs/1.0.0.RC1/reference/html/#repositories.custom-implementations

Reply:
I really like that approach, it works really well!